frontend.sh

```diff
@@ -93,6 +93,17 @@ build() {
   fi
 }
 
+# Run tasks to build all project assets for production distribution.
+dry_run() {
```

Comment:
The tox environment works beautifully. Can we remove this function, or have it call `tox -e validate-assets` (and rename it if we're keeping it)?

Reply:
fixed in https://github.com/cfpb/cfgov-refresh/pull/3537/commits/70d709ce9ce0ce058da1cfa9761914817d8341ca.